docs/source/contributing/review_pr_pymc_examples.md

```diff
@@ -0,0 +1,113 @@
+(reveiw_pr_pymc_examples)=
+# Review a PR on pymc-examples
+
+The target audience for this page are reviewers of PRs in pymc-examples
+repo. It mostly gathers resources from its
+[contributing guide](https://github.com/pymc-devs/pymc-examples/blob/main/CONTRIBUTING.md),
+[PR template](https://github.com/pymc-devs/pymc-examples/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+and the {ref}`jupyter_style` page to
+centralize all resources for reviewers and put special emphasis on
+reviewer tasks and responsibilities.
+
+:::{important}
+The most important guideline is the following: **When you aren't _completely_
+sure about something say so, ask around and (re)read the documentation**.
```

Comment:
Delete "say so"

Reply:
doesn't that imply that "leave a comment saying I don't know this and therefore won't review it" is not an option? I would like to keep something with that message, not necessarly "say so" though, there are probably more clear and formal options I didn't think about.

Reply:
```suggestion
sure about something communicate it, ask around and (re)read the documentation**.
```